tel-search-connector/config/variables.yaml

```diff
@@ -6,4 +6,6 @@
 # '<project>/Config/_<environment>/variables.yaml
 #
 Variables:
-#  myVariable: value
+  # Get your key for unlimited access @ https://tel.search.ch/api/getkey
+  # [password]
+  tel.search.api.key: "12345_EXAMPLE_KEY"
```

Comment:
I think we should leave it empty -> key: ""
So that we do not enable this feature and therefore modify the URI even when no real key is available